packages/amplify-util-mock/src/utils/index.ts

```diff
@@ -45,3 +45,7 @@ function isUnsupportedJavaVersion(stderr: string | null): boolean {
 }
 
 export const _isUnsupportedJavaVersion: (stderr: string | null) => boolean = isUnsupportedJavaVersion;
+
+export const isWindowsPlatform = () => {
+  return process?.platform?.startsWith('win');
+}
```

Comment:
I think this check exists elsewhere, can you check that this isn't a duplicate function?

If you need to keep it, please add the missing return type